ToDoWebApi/Applications/UserOperations/Commands/CreateUser/CreateUserCommandValidator.cs

```diff
@@ -0,0 +1,15 @@
+using FluentValidation;
+
+namespace ToDoWebApi.Applications.UserOperations.Commands.CreateUser
+{
+    public class CreateUserCommandValidator : AbstractValidator<CreateUserCommand>
+    {
+        public CreateUserCommandValidator()
+        {
+            RuleFor(command => command.Model.Name).NotEmpty();
+            RuleFor(command => command.Model.Surname).NotEmpty();
```

Comment:
Name ve Surname için minimum 2 karakter gibi bir kontrol de eklenebilir bence.

Reply:
Eklendi.